packages/salesforcedx-vscode-core/src/messages/i18n.ts

```diff
@@ -58,19 +58,20 @@ export const messages = {
   parameter_gatherer_invalid_forceide_url:
     "The forceide:// URL is invalid. From your subscriber's org, copy and paste the forceide:// URL shown on the Apex Debugger page in Setup.",
 
-  force_org_create_default_scratch_org_text: 'SFDX: Create a Default Org...',
+  force_org_create_default_scratch_org_text:
+    'SFDX: Create a Default Scartch Org...',
```

Comment:
There's a typo here: `Scartch` instead of `Scratch`.

Reply:
Just saw it, fixing!